.travis.yml

```diff
@@ -42,15 +43,16 @@ script:
   # Setup redis
   - docker pull redis:alpine
   - docker run -d --name redis redis:alpine
-  - timeout $GREP_TIMEOUT grep -m 1 'The server is now ready to accept connections on port 6379' <(docker logs --follow redis 2>&1)
+  - timeout $GREP_TIMEOUT grep -m 1 'Ready to accept connections' <(docker logs --follow redis 2>&1)
 
   # Build RapidPro and run it
   - docker pull rapidpro/rapidpro-base
+  - docker pull "$IMAGE"
   - >
-    docker build --tag "$IMAGE" \
+    docker build --pull --cache-from "$IMAGE" \
+      --tag "$IMAGE" \
       --build-arg RAPIDPRO_VERSION=$VERSION \
-      --build-arg VCS_REF=$TRAVIS_COMMIT \
-      --build-arg BUILD_DATE=$(date -Iseconds) \
+      --build-arg RAPIDPRO_REPO=$REPO \
```

Comment:
Removing the VCS_REF & BUILD_DATE arguments prevents the cache from invalidating, faster builds